templates/user/dashboard/navbar.tmpl

```diff
@@ -75,30 +75,28 @@
 		{{end}}
 
 	{{if .ContextUser.IsOrganization}}
-		<div class="right stackable menu">
```

Comment:
This doesn't seem related to your fix?

(Or your screenshots aren't showing it)